libc-test/build.rs

```diff
@@ -3309,6 +3310,8 @@ fn test_linux(target: &str) {
             | "IFLA_PERM_ADDRESS"
             | "IFLA_PROTO_DOWN_REASON"
             if sparc64 => true,
+            // Added in Linux 5.13
+            "PTRACE_GET_RSEQ_CONFIGURATION" => true,
```

Comment:
If it's added in Linux 5.13, we could do testing except for sparc64 as Ubuntu 22.04 uses 5.15 kernel version.